btf-utils/btfhub-btfgen.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash -e
```

Comment:
Where is this script called?

Reply:
It's not called anywhere, I made it to download btfhub-archive and generate the reduced files

Reply:
How would we ship it with our container? It should be called at docker build time tho right?

Reply:
This was meant to run everytime someone changed system_monitor.c file. I was going to store the reduced files as a zip and unzip it in the container

Reply:
Wouldn't generating it as part of container image cover the case for 
> run everytime someone changed system_monitor.c file

We can assume that no one would expect BTF files when using non CORE in development environment.

Reply:
Generating it in the container means having btfhub-archive, which isn't feasible cause it takes around 5-ish GB

Reply:
Hey @AllMight2099 , We have a misunderstanding here.

I am saying it to generate it at build time, not runtime. Like how we compile the KubeArmor code to a single binary in our Dockerfile and ship only the binary.

Similarly we download and reduce the btfhub at build time and copy the reduced files from staging to main container using Dockerfile.

Reply:
Ohhh, got it

Reply:
For reference

https://github.com/kubearmor/KubeArmor/blob/fc08819ea1efc8d3c68b399d7b73fe44859baab6/Dockerfile#L18 built in builder image

https://github.com/kubearmor/KubeArmor/blob/fc08819ea1efc8d3c68b399d7b73fe44859baab6/Dockerfile#L32
Copied from builder image to actual image.

Reply:
I'll make those changes